helm/url-redirector/values.yaml

```diff
@@ -0,0 +1,113 @@
+# Default values for url-redirector.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 3
+
+image:
+  repository: registry.gitlab.com/bobymcbobs/url-redirector
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext:
+  readOnlyRootFilesystem: true
+  privileged: false
+  runAsUser: 1000
+  runAsGroup: 1000
+  allowPrivilegeEscalation: false
+
+service:
+  type: ClusterIP
+  port: 8080
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: url-redirector.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+
+env:
+  - name: TZ
+    value: "Pacific/Auckland"
+  - name: APP_PORT
+    value: ":8080"
+  - name: APP_CONFIG_YAML
+    value: /app/cfg/config.yaml
+
+appConfig: |
```

Comment:
nit: This would be nicer as just yaml without the string block (`|`)

Reply:
I can't figure out the yaml to make that work.  Lets work on that in a follow up change.

Reply:
No stress, sounds good!